src/pages/service.tsx

```diff
@@ -152,7 +152,6 @@ const ServicePage: React.FC = () => {
     deviceFromHash.includes("ios") ||
     deviceFromHash.includes("iphone") ||
     deviceFromHash.includes("ipad")
-  const hideMessenger = deviceFromHash.includes("pc") || deviceFromHash.includes("mac")
 
   // window.console.log(hideTeams, hideMessenger, deviceFromHash, OS)
```

Comment:
Comment

Reply:
Is removed in the teams PR, so will leave it in here.